larpandoracontent/LArControlFlow/NeutrinoIdTool.h

```diff
@@ -10,11 +10,22 @@
 
 #include "larpandoracontent/LArControlFlow/MasterAlgorithm.h"
 
+#include "larpandoracontent/LArObjects/LArSupportVectorMachine.h"
+
+#include <functional>
+
 namespace lar_content
 {
 
 /**
  *  @brief  NeutrinoIdTool class
+ *
+ *  Compares the neutrino and cosmic hypotheses of all of the slices in the event. Uses an SVM to calculate the probability of each slice 
+ *  containing a neutrino interaction. The N slices with the highest probabilities are identified as a neutrino (if sufficiently probable)
+ *  all other slices are deemed cosmogenic.
+ *
+ *  If training mode is switched on, then the tool will write SVM training exmples to the specified output file. The events selected for 
+ *  training must pass (user congigurable) slicing quality cuts. Users may also select events based on their interaction type (nuance code).
```

Comment:
Typo: "congigurable"

Is this all correctly formatted to be picked up by doxygen? It probably is, although I normally add n sets of four spaces (no tab characters) to indent so that the left text margin is in line the NeutrinoIdTool class. Some people add html commands, but we have never done this in Pandora.